src/interfaces/validator/validate-update.js

```diff
@@ -0,0 +1,110 @@
+/**
+*
+* @licstart  The following is the entire license notice for the JavaScript code in this file.
+*
+* RESTful API for Melinda - record validation services
+*
+* Copyright (C) 2023 University Of Helsinki (The National Library Of Finland)
+*
+* This file is part of melinda-rest-api-validator
+*
+* melinda-rest-api-validator program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* melinda-rest-api-validator is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @licend  The above is the entire license notice
+* for the JavaScript code in this file.
+*
+*/
+
+import createDebugLogger from 'debug';
+//import {MarcRecord} from '@natlibfi/marc-record';
+
+const debug = createDebugLogger('@natlibfi/melinda-rest-api-validator:validator:validate-update');
+const debugData = debug.extend('data');
+
+// Note: when we get new recordImport catalogers or imported conversions, these need to be updated
+// These could also exist as env-configs
+const catalogerToConversionNameMapping = {
+  'IMP_HELMET': 'Helmet to Melinda MARC transformation',
+  'IMP_TATI': 'TATI to Melinda MARC transformation',
+  'IMP_ENNAKK': 'ONIX3 to MARC transformation',
+  'IMP_VPKPL': 'ONIX3 to MARC transformation',
+  // testing mapping, remove later
+  'MINTTU': 'MINTTU 884 test'
```

Comment:
This MINTTU mapping could be more generic. Maybe 'TEST: 884 test transformation'?

Reply:
Removes